src/argilla/server/telemetry.py

```diff
@@ -64,9 +67,14 @@ def __post_init__(self, enable_telemetry: bool, disable_send: bool, api_key: str
             "platform": platform.platform(),
             "python_version": platform.python_version(),
             "sys_version": platform.version(),
+            "deployment": "quickstart" if is_quickstart_server() else "server",
+            "docker": "yes" if is_running_on_docker_container() else "no",
```

Comment:
It's not supported to send a boolean value? like:
```suggestion
            "docker": is_running_on_docker_container(),
```